org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/completion/LSContentAssistProcessor.java

```diff
@@ -279,15 +279,15 @@ private static List<ICompletionProposal> toProposals(IDocument document, int off
 	public IContextInformation @Nullable [] computeContextInformation(ITextViewer viewer, int offset) {
 		IDocument document = viewer.getDocument();
 		if (document == null) {
-			return new IContextInformation[] { /* TODO? show error in context information */ };
+			return null; /* TODO? show error in context information */
```

Comment:
this changes to return null seem to make the code more inline with the documentation of the method we are overriding, and I think it would be nice to have a separate PR for them that we can merge independently.